File: src/main/java/com/kairlec/local/utils/RequestUtils.java
package com.kairlec.local.utils;

import javax.servlet.http.HttpServletRequest;

public abstract class RequestUtils {
    private RequestUtils() {
        throw new AssertionError("工具类不允许实例化");
    }

    public static String getSourcePath(HttpServletRequest request) {
        return request.getParameter("source");
    }

    public static String getTargetPath(HttpServletRequest request) {
        return request.getParameter("target");
    }

}

File: src/main/java/com/kairlec/local/utils/RequestUtils.kt
package com.kairlec.local.utils

import javax.servlet.http.HttpServletRequest;

object RequestUtils {
    fun getSourcePath(request: HttpServletRequest): String? {
        return request.getParameter("source")
    }

    fun getTargetPath(request: HttpServletRequest): String? {
        return request.getParameter("target")
    }
}
